lib/hipe/doc/src/hipe_app.xml

```diff
@@ -47,6 +47,71 @@
       Details on HiPE compiler options are given by <c>hipe:help_options()</c>.</p>
   </description>  
   <section>
+    <title>Feature Limitations</title>
+    <p>
+      The HiPE compiler is in general compliant with the normal BEAM compiler,
+      with respect to semantic behavior. There are however features in the BEAM compiler
+      and the runtime system that have limited or no support for HiPE compiled modules.
+    </p>
+    <taglist>
+      <tag>Stack traces</tag>
+      <item><p>Stack traces returned from <seealso marker="erts:erlang#get_stacktrace/0">
+      <c>erlang:get_stacktrace/0</c></seealso> or as part of <c>'EXIT'</c> terms
+      can look incomplete if HiPE compiled functions are involved. Typically a stack trace
+      will contain only BEAM compiled functions or only HiPE compiled functions, depending
+      on where the exception was raised.</p>
+      <p>Source code line numbers in stack traces are also not supported by HiPE compiled functions.</p>
+      </item>
+
+      <tag>Tracing</tag>
+      <item><p>Modules compiled with HiPE have very limited support for runtime tracing
+      using <seealso marker="erts:erlang#trace_pattern/3"><c>erlang:trace_pattern/3</c></seealso>.</p>
+      </item>
```

Comment:
Can you elaborate?

Reply:
I think all types of call tracing lack support. You may try to enable call trace for hipe compiled modules but you will not get any trace messages from them.

Reply:
That'd certainly explain the test failures I observe in my tracer... Thanks.